pkg/volume/flexvolume/plugin.go

```diff
@@ -42,9 +42,30 @@ type flexVolumePlugin struct {
 	unsupportedCommands []string
 }
 
-var _ volume.AttachableVolumePlugin = &flexVolumePlugin{}
+type flexVolumeAttachablePlugin struct {
+	flexVolumePlugin
+}
+
+var _ volume.AttachableVolumePlugin = &flexVolumeAttachablePlugin{}
 var _ volume.PersistentVolumePlugin = &flexVolumePlugin{}
 
+func (plugin *flexVolumeAttachablePlugin) isAttachable() (bool, error) {
+	call := plugin.NewDriverCall(initCmd)
+	res, err := call.Run()
+	if err != nil {
+		return false, err
+	}
+
+	// By default all plugins are attachable, unless they report otherwise.
+	cap, ok := res.CapabilityMap[attachCapability]
```

Comment:
I want to rename it to capabilities, similar to annotations & labels. Will update the PR.